app/Http/Controllers/SrdController.php

```diff
@@ -40,12 +40,19 @@ public function searchRoutes(Request $request): JsonResponse
             })
                 ->where('maximum_level', '>=', $requestData['requestedLevel']);
         }
+        
+        if (strlen($requestData['destination']) > 4) {
+            $exit = $requestData['destination'];
+        } 
+        else {
+            $exit = null; 
+        }
 
         // Format the results
         $results = $query->get()->map(function (SrdRoute $route) {
             $routeString = is_null($route->sid)
                 ? $route->route_segment
-                : sprintf('%s %s', $route->sid, $route->route_segment);
+                : sprintf('%s %s', $route->sid, $route->route_segment, $exit);
```

Comment:
To access`$exit` in this callback, you need to use it above:

e.g. `function (SrdRoute $route) use ($exit) {`